chart/templates/deployment.yaml

```
@@ -28,6 +28,16 @@ spec:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            - name: file_name
+              value: {{ .Values.file.name }}
+            - name: database_expose
+              value: {{ .Values.database.expose }}
+            - name: cycle_time
+              value: {{ .Values.cycle.time }}
+            - name: use_mock_data
+              value: {{ .Values.use.mock.data }}
+              
```

```suggestion
          env:
            - name: cycle_time
              value: {{ .Values.cycle_time_s }}
            {{- if .Values.file_name }}
            - name: file_name
              value: {{ .Values.file_name | quote  }}
            {{- end }}
            {{- if .Values.database_expose }}
            - name: database_expose
              value: {{ .Values.database_expose | quote  }}
            {{- end }}
            {{- if .Values.use_mock_data }}
            - name: use_mock_data
              value: {{ .Values.use_mock_data | quote  }}
            {{- end }}
              
```
If you change the naming as i suggest then change it here as well. I added the "quote" as it needs it in my experience. Furthermore by doing like this they are only set if they are in the values-file.